tests/phpunit/integration/class-test-case.php

```diff
@@ -33,7 +33,7 @@ public function setUp() {
 		parent::setUp();
 		Monkey\setUp();
 
-		if ( $this->was_reset ) {
+		if ( ! $this->was_reset ) {
```

Comment:
Just to note, the setUp() resets now run for (all) 50 integration tests in the Fields API and 8 integration tests in the Filters API.
The tearDown() resets run for 18 integration tests in the Fields API and (the same) 8 integration tests in the Filters API